dotcom-rendering/src/web/components/AdSlot.tsx

```diff
@@ -388,21 +355,21 @@ export const AdSlot: React.FC<Props> = ({ position, display }) => {
 						// 1. file mark: 432b3a46-90c1-4573-90d3-2400b51af8d0
 						// 2. file mark: c66fae4e-1d29-467a-a081-caad7a90cacd
 						data-tablet={[
-							`${Size.outOfPage}`,
-							`${Size.empty}`,
-							`${Size.fabric}`,
-							`${Size.fluid}`,
-							`${Size.leaderboard}`,
+							`${adSizes.outOfPage.toString()}`,
+							`${adSizes.empty.toString()}`,
+							`${adSizes.fabric.toString()}`,
+							`${adSizes.fluid.toString()}`,
+							`${adSizes.leaderboard.toString()}`,
 						].join('|')}
 						data-desktop={[
-							`${Size.outOfPage}`,
-							`${Size.empty}`,
-							`${Size.leaderboard}`,
+							`${adSizes.outOfPage.toString()}`,
+							`${adSizes.empty.toString()}`,
+							`${adSizes.leaderboard.toString()}`,
 							`940,230`,
 							`900,250`,
```

Comment:
Possibly outside the scope of this PR, but it would be cool to understand why these values are hardcoded here instead of being given proper names and referenced like the others. Do we need to support these sizes? If so, should we move them to [ad-sizes](https://github.com/guardian/commercial-core/blob/55cdb59619dfce1a61aba3c7b12913f6c5d78501/src/ad-sizes.ts) in `commercial-core`?

Reply:
Very good question 🤔 

I dug into this some and ended up on [this PR](https://github.com/guardian/dotcom-rendering/pull/2258) which seems to be where the `Size` abstraction was introduced. That abstraction wasn't arbitrary though, it was built based on [this file](https://github.com/guardian/frontend/blob/main/static/src/javascripts/projects/commercial/modules/ad-sizes.js) in Frontend but as you can see, that file has no abstraction for these two sizes. So rather than change then or create a divergence from the legacy code, they were left hard coded.

That still doesn't explain why they are there though. So I dug further to find the original place where they were added and found they were sourced from [this code ](https://github.com/guardian/frontend/blob/9d2ee56372de4bb730146100527f83b2efd41c35/common/app/views/support/Commercial.scala#L142)in Frontend.

So the issue is/was that the named abstraction originally defined in Frontend, that we reverently copied over to DCR, was not complete and so when we tried to match the two up, we had this size left over.

Should it be added?
Yes. I think so. DCR is now rendering these slots 100% so there's not a problem with divergence anymore.